homeassistant/components/bmp280/sensor.py

```diff
@@ -0,0 +1,158 @@
+"""Platform for Bosch BMP280 Environmental Sensor integration."""
+from datetime import timedelta
+import logging
+
+from adafruit_bmp280 import Adafruit_BMP280_I2C
+import board
+from busio import I2C
+import voluptuous as vol
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    PLATFORM_SCHEMA,
+)
+from homeassistant.const import CONF_NAME, PRESSURE_HPA, TEMP_CELSIUS
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = "BMP280"
+DEFAULT_SCAN_INTERVAL = timedelta(seconds=15)
```

Comment:
This constant isn't used. To define the default scan interval for platforms please define the `SCAN_INTERVAL` constant at the module level.